### PR TITLE
Adds Ambassador integration for Centraldashboard

### PR DESCRIPTION
**Pull request by swiftdiaries**
Addresses issue: https://github.com/kubeflow/kubeflow/issues/528

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/529)

### CODE REVIEW

File: kubeflow/core/centraldashboard.libsonnet [hunk 1]
```diff
@@ -103,6 +88,9 @@ all(params):: [
       apiVersion: "v1",
       kind: "ServiceAccount",
       metadata: {
+        labels: {
+          app: "centraldashboard",
```
Comment (ankushagarwal) [inline kubeflow/core/centraldashboard.libsonnet]: Can you change this to name: "centraldashboard" as well since you changed the service

**Comment by ankushagarwal** [comment]
Can you add some tests for centraldashboard.libsonnet in kubeflow/core/tests

**Comment by swiftdiaries** [comment]
cc @ankushagarwal

File: OWNERS [hunk 2]
```diff
@@ -6,7 +6,9 @@ approvers:
   - gaocegege
   - jlewi
   - lluunn
+  - pdmack
```
Comment (inc0) [inline OWNERS]: need to fix rebase:)

File: README.md [hunk 3]
```diff
@@ -71,17 +71,23 @@ In order to quickly set up all components, execute the following commands:
...
 ks init ${APP_NAME}
 cd ${APP_NAME}
 ks env set default --namespace ${NAMESPACE}
 
 # Install Kubeflow components
-ks registry add kubeflow github.com/kubeflow/kubeflow/tree/master/kubeflow
-ks pkg install kubeflow/core
-ks pkg install kubeflow/tf-serving
-ks pkg install kubeflow/tf-job
+ks registry add kubeflow github.com/kubeflow/kubeflow/tree/${VERSION}/kubeflow
```
Comment (inc0) [inline README.md]: ditto

**Comment by ankushagarwal** [comment]
This change places the central dashboard at the path `/ui`. Can we have it at the path`/`? Then this becomes the _homepage_ for kubeflow. Currently the _homepage_ 404s since there is nothing there

**Comment by swiftdiaries** [comment]
Changed centraldashboard to be the homepage.

**Comment by ankushagarwal** [comment]
@swiftdiaries This commit seems to add a lot of unrelated changes : https://github.com/kubeflow/kubeflow/commit/5c91328f5056fb10dc033f0f3e778b7198aa22c4

Can you fix that?

**Comment by swiftdiaries** [comment]
Closing due to CLA issues.

**Pull request closed without merging**